crates/core/src/db/relational_db.rs

```diff
@@ -924,6 +1097,9 @@ pub mod tests_utils {
         /// Note that flushing the log is an asynchronous process. [`Self::reopen`]
         /// ensures all data has been flushed to disk before re-opening the
         /// database.
+        ///
+        /// Requires access to a Tokio runtime in order to spawn a snapshot worker.
```

Comment:
Do we want or need to start two runtimes? `durable` could be passed in a handle, or the handle can be obtained from the `TestDB`.